js/anxpro.js

```diff
@@ -451,7 +826,99 @@ module.exports = class anxpro extends Exchange {
+        //     tradedCurrency: 'XRP',
+        //     settlementCurrency: 'BTC',
+        //     tradedCurrencyAmount: '400.00000000',
+        //     buyTradedCurrency: true,
+        //     limitPriceInSettlementCurrency: '0.00007129',
+        //     timestamp: '1522547850000',
+        //     orderId: '62a8be4d-73c6-4469-90cd-28b4726effe0',
+        //     tradedCurrencyAmountOutstanding: '0.00000000',
+        //     orderStatus: 'FULL_FILL',
+        //     executedAverageRate: '0.00007127',
+        //     trades:
+        //     [ { tradeId: 'fe16b796-df57-41a2-b6d9-3489f189749e',
+        //         orderId: '62a8be4d-73c6-4469-90cd-28b4726effe0',
+        //         timestamp: '1522547850000',
+        //         tradedCurrencyFillAmount: '107.91298639',
+        //         settlementCurrencyFillAmount: '0.00768772',
+        //         settlementCurrencyFillAmountUnrounded: '0.00768772',
+        //         price: '0.00007124',
+        //         ccyPair: 'XRPBTC' },
+        //         { tradeId: 'e2962f67-c094-4243-8b88-0cdc70a1b1c7',
+        //             orderId: '62a8be4d-73c6-4469-90cd-28b4726effe0',
+        //             timestamp: '1522547851000',
+        //             tradedCurrencyFillAmount: '292.08701361',
+        //             settlementCurrencyFillAmount: '0.02082288',
+        //             settlementCurrencyFillAmountUnrounded: '0.02082288',
+        //             price: '0.00007129',
+        //             ccyPair: 'XRPBTC' } ] }
+        const tradedCurrency = this.safeString (order, 'tradedCurrency');
+        const orderStatus = this.safeString (order, 'orderStatus');
+        let status = undefined;
+        if (orderStatus === 'ACTIVE')
+            status = 'open';
+        else if (orderStatus === 'FULL_FILL')
+            status = 'closed';
+        else if (orderStatus.indexOf ('CANCEL') >= 0)
+            status = 'canceled';
+        const settlementCurrency = this.safeString (order, 'settlementCurrency');
+        const symbol = this.findSymbol (tradedCurrency + '/' + settlementCurrency);
+        const buyTradedCurrency = this.safeString (order, 'buyTradedCurrency');
+        const side = buyTradedCurrency === 'true' ? 'buy' : 'sell';
+        const timestamp = this.safeInteger (order, 'timestamp');
+        let lastTradeTimestamp = undefined;
+        const trades = [];
+        let filled = 0;
+        const type = this.safeString (order, 'orderType').toLowerCase ();
+        for (let i = 0; i < order['trades'].length; i++) {
+            const trade = order['trades'][i];
+            const tradeTimestamp = this.safeInteger (trade, 'timestamp');
+            if (!lastTradeTimestamp || lastTradeTimestamp < tradeTimestamp)
+                lastTradeTimestamp = tradeTimestamp;
+            const parsedTrade = this.extend (this.parse_v3_Trade (trade), { 'side': side, 'type': type });
+            trades.push (parsedTrade);
+            filled += parsedTrade['amount'];
```

Comment:
↑ We need to use `this.sum` for reasons explained here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#adding-numbers-and-concatenating-strings